### ISSUE TITLE
Unknown event handler property `onTouchTap`

### ISSUE THREAD

**Issue by filoxo** [ukrbublik/react-copy-html-to-clipboard#1]
I just installed this in a project and started seeing this warning. React 15.6.1

```
Warning: Unknown event handler property `onTouchTap`. It will be ignored.
    in button (created by AllTemplatesContainer)
    in CopyToClipboard (created by AllTemplatesContainer)
    in cps-tooltip (created by i)
    in i (created by AllTemplatesContainer)
    in span (created by PreviewLite)
```

Looks like this commit https://github.com/ukrbublik/react-copy-html-to-clipboard/commit/e3c469d11f360287f0d9cefb4333ac26a0baba83#diff-25d902c24283ab8cfbac54dfa101ad31 added support for onTouchTap but enforced it for all users, instead of just those that are using `react-tap-event-plugin`. Perhaps this should an an opt-in feature? I could submit a PR but wanted to have a quick discussion as to where to go with a fix.

**Comment by filoxo** [comment]
Thoughts on this?

**Comment by ukrbublik** [comment]
Hi. Sorry, I haven't seen your first message.
How do you want to remove this warning?
You can submit PR.

**Comment by filoxo** [comment]
At the time, I was going to offer to fix this but after digging in to the specifics of `onTouchTap`, I realized its not worthwhile. onTouchTap is no longer supported with React 16.4+ (see [issue #121 for react-tap-event-plugin](https://github.com/zilverline/react-tap-event-plugin/issues/121)), and that plugin is no longer a dependency for material-ui (see [issue #7129 for material-ui](https://github.com/mui-org/material-ui/issues/7129)). Instead, the commit I referenced should simply be undone. Since that would be a breaking change albeit a simple one, I'll let you decide how you want to do that. Thanks for the response!
